internal/slacklog/message.go

```diff
@@ -453,6 +454,26 @@ func (f *MessageFile) DownloadFilename(url, suffix string) string {
 	return filename
 }
 
+// 'text/plain' な添付ファイルをHTMLに埋め込む
+// 存在しない場合、エラーを表示する
+func (f *MessageFile) CodeHTML() (string, error) {
```

Comment:
```suggestion
// CodeHTML: 'text/plain' な添付ファイルをHTMLに埋め込む
// 存在しない場合、エラーを表示する
func (f *MessageFile) CodeHTML() (string, error) {
```


`CodeHTML` という名前の関数のコメントは golint によれば `CodeHTML` で始めることになっています。
英語ならばこんな感じ

```
// `CodeHTML genarates `<code>` tag with ...
```

Reply:
`:` の前にスペースが必要なはずですね